README.md

```diff
@@ -0,0 +1,12 @@
+Index a file of records without using the database or hardcoded filesystem paths:
+
+```bash
+
+docker-compose build
+docker-compose run --rm traject bundle install
+docker-compose run --rm traject bundle exec fetch_new_hlb ./lib/translation_maps
+# get some sample records somehow
+docker-compose run traject bundle exec bin/tindex localhost examples/sample_records.json.gz
+# ensure documents are committed
+curl -v http://localhost:9033/solr/catalog/update?commit=true
```

Comment:
I'm not sure this is necessary, but when I indexed 10,000 sample records I did not immediately see them in solr.

Reply:
n/m I see the `commit` call in `fullindex`, `catchup_since`, and `index_date`